streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -664,16 +664,16 @@ private void initializeTaskTime(final Map<TopicPartition, OffsetAndMetadata> off
     }
 
     @Override
-    public Map<TopicPartition, Long> purgableOffsets() {
-        final Map<TopicPartition, Long> purgableConsumedOffsets = new HashMap<>();
+    public Map<TopicPartition, Long> purgeableOffsets() {
```

Comment:
Not taking a hard stance on this spelling, just aiming for consistency across the code base